app/models/cohort_list_row.rb

```diff
@@ -0,0 +1,44 @@
+class CohortListRow
+  include ActiveModel::Model
+
+  attr_accessor :parent_name,
+                :parent_relationship,
+                :parent_email,
+                :parent_phone,
+                :child_first_name,
+                :child_last_name,
+                :child_common_name,
+                :child_date_of_birth,
+                :child_address_line_1,
+                :child_address_line_2,
+                :child_address_town,
+                :child_address_postcode,
+                :child_nhs_number
+
+  validates :parent_name, presence: true
```

Comment:
I think we should replicate the length validations here. Even if it's (slightly) less susceptible to attack, it'd be a good idea to just keep them lined up.

Reply:
Just had a thought ... the length validations will kick in later, when trying to save the record wherever these are destined to go ... so maybe it's not that crucial to have length validations here actually.